SCClassLibrary/Common/Files/File.sc

```diff
@@ -18,6 +18,11 @@ File : UnixFILE {
 		file = this.new(pathName, mode);
 		^{ function.value(file) }.protect({ file.close });
 	}
+	*read { arg pathName, mode = "r", selector = \readAllString;
```

Comment:
yes, it's a notion that comes from the smalltalk tradition. A symbol selects a method from the receiving object's vocabulary.